.circleci/config.yml

```diff
@@ -976,19 +976,19 @@ jobs:
     - run:
         name: Build Packages
         command: |
-          ./tools/releases/distros.sh --package --version $CIRCLE_TAG --sha $CIRCLE_SHA1
+          ./tools/releases/distros.sh --package --version $(make build/version) --sha $CIRCLE_SHA1
     - run:
         name: Push Packages
         command: |
-          ./tools/releases/distros.sh --release --version $CIRCLE_TAG
+          ./tools/releases/distros.sh --release --version $(make build/version)
     - run:
         name: Build Docker
         command: |
-          ./tools/releases/docker.sh --build --version $CIRCLE_TAG
+          ./tools/releases/docker.sh --build --version $(make build/version)
```

Comment:
I guess this is to minimize the change but does it make sense rebuilding everything when it was already built in the parent jobs?

Reply:
Let's try it on separate PR. It will be built on master/release only anyways, so it won't slow down CI on PRs